### ISSUE TITLE
[webapp] Container update command doesn't validate image name

### ISSUE THREAD

**Issue by lostintangent** [Azure/azure-cli#1337]
I just noticed that the new `Docker Container` pane for App Service on Linux instances provides image name validation, which is a really handy feature for preventing subtle config errors:

<img width="450" alt="screen shot 2016-11-15 at 4 08 20 pm" src="https://cloud.githubusercontent.com/assets/116461/20329709/da22b6a6-ab4e-11e6-8ee7-274f9bbd0104.png">

It would be great if the `az appservice web config container update` command provided the same validation, so that regardless whether you were managing your app from the CLI or the portal, you receive the same experience. Definitely not a blocker, but I prefer the CLI-driven workflow, and therefore, would love to see parity with the portal as applicable.

**Comment by lostintangent** [comment]
As another point of discussion, I just spoke with someone who was running into issues deploying a container to App Service, and the problem was a mis-typed container name, that the CLI didn't validate. This feature request would definitely help guide folks towards success :)

**Comment by lostintangent** [comment]
Additionally, the CLI doesn't validate the server URL when pulling an image from a private registry. It requires you to include the scheme in the URL, but you can update with `foo.com` as opposed to `https://foo.com` and it will happily accept it. The portal, however, provides a helpful error to fix this.

**Comment by yugangw-msft** [comment]
It is bit more coding involved, that we need to check it is a v1 or v2 registry and use appropriate API to detect.

**Comment by ahmedelnably** [comment]
It would be nice to have an api from the backend to validate the image

**Comment by panchagnula** [comment]
CC: @btardif to follow up on the API support for full validation.

**Comment by btardif** [comment]
This WI if blocked on API Support for both Windows and Linux container scenarios.

**Comment by panchagnula** [comment]
@btardif do you know if we have an ETA on the API availability here?

**Comment by btardif** [comment]
@panchagnula  Backend is targeting ANT85 so moving this out to a more appropriate sprint

**Comment by panchagnula** [comment]
we will assign a dev to work on this once the backend work to support this is complete.

**Comment by panchagnula** [comment]
Need to move this since we are waiting for a backend API work to be completed for this.

**Comment by panchagnula** [comment]
@btardif  do you know if this now available?

**Comment by panchagnula** [comment]
@btardif please assign this back to me when the API is ready for use. Thanks!

**Comment by panchagnula** [comment]
@btardif just checking if this change is ready and deployed for consumption? Thank you.

**Comment by panchagnula** [comment]
@btardif let me know when this is ready.

**Comment by btardif** [comment]
Closing this issue as won't fix. 

Validating a container image in most cases will require permissions on the container repository that under secure environments would only be available to the runtime and not the operator.
